tools/packager/Makefile

```diff
@@ -189,8 +189,9 @@ $(BUILD_TARGET_ROOT)/docker/Dockerfile: docker/Dockerfile
 	cp $< $@
 
 $(BUILD_TARGET_ROOT)/docker.tagged: $(BUILD_TARGET_ROOT)/docker/Dockerfile $(BUILD_TARGET_ROOT)/docker/share/marathon
-	cd $(BUILD_TARGET_ROOT)/docker && docker build --build-arg MESOS_PKG_VERSION=$(MESOS_PKG_VERSION) --tag mesosphere/marathon:v$(PKG_VER) . 2>&1 | tee "docker.work"
-	mv $(BUILD_TARGET_ROOT)/docker/docker.work $@
+	mkdir -p "$(dir $@)"
+	cd $(BUILD_TARGET_ROOT)/docker && docker build --build-arg MESOS_PKG_VERSION=$(MESOS_PKG_VERSION) --tag mesosphere/marathon:v$(PKG_VER) . 2>&1 | tee "$(PWD)/$@.work"
```

Comment:
I'm not confident this'll work. Isn't `PWD` == `$(BUILD_TARGET_ROOT)/docker` and then `$@.work` expands to `$(BUILD_TARGET_ROOT)/docker`?